### PR TITLE
Remove Jsonata - Mourning the Generics

### PR DESCRIPTION
**Pull request by M-Kusumgar**
## Problems/motivation:
- Input time series was taking too long to update the graphs because of jsonata being slow

## New features/solutions:
- Stepper connectors are more centered
- Download button icon is now aligned to the center
- Jsonata has been removed and replaced with base typescript - please compare the plotly.vue file change with https://github.com/mrc-ide/hint/blob/master/src/app/src/main/resources/metadata/input-time-series-config-jsonata.txt and chekc that they match. For the most part I have gone line by line, there may be optimisation to be done but, for this PR, I just aim to "transpile" from jsonata to typescript
- Integration tests have been moved to unit testing for plotly

## How PR should behave when tested:
- Input time series should be fast when switching between any of the filters

## How to run app:
1. Run the app according to the README

## Note
I was thinking that renaming everything and removing the endpoint & dealing with state stuff can be in another PR to help isolate this slightly more complicated diff (tickets: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4501/Rename-Generics-to-InputTimeSeries, https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4502/Remove-jsonata-metadata-in-state-and-the-backend)? Maybe after the packages PR (the one which will replace the vue-next-dynamic-form and vue-charts package folders in Hint)?

### CODE REVIEW

**Comment by M-Kusumgar** [comment]
ignore codecov, I haven't deleted any tests, we have the same amount of testing as before so should be the same coverage

**Review by LekanAnni — CHANGES_REQUESTED** [review]
Looking good,

* Looks like test coverage for this PR should be covered as they are actually real 😆
* Can you reduce use of `any` data types ?
* Overall, the functionality is satisfactory; however, there seems to be a minor issue with the presentation of the data using Benin data, as it appears slightly compressed.

I don't think I have done justice reviewing your PR, this one rattled me but I have added a few inline comments. PR also works

File: src/app/static/src/app/components/genericChart/Plotly.vue
Comment (M-Kusumgar) [reply src/app/static/src/app/components/genericChart/Plotly.vue:20]: done

File: src/app/static/src/app/components/genericChart/Plotly.vue
Comment (M-Kusumgar) [reply src/app/static/src/app/components/genericChart/Plotly.vue:48]: removed

**Review by EmmaLRussell — APPROVED** [review]
As discussed: happy to leave it til future tickets to deal with renaming from GenericChart, and to refactor the monolithic `getData` method into something more digestible (could you create this ticket if it doesn't exist already?).

**Comment by M-Kusumgar** [comment]
> refactor the monolithic `getData` method into something more digestible (could you create this ticket if it doesn't exist already?).

done, https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4524/Refactor-the-getData-computed-value-into-nicer-code

**Comment by M-Kusumgar** [comment]
in the interest of time, will be merging this in since it has one approval, but can look over this after merge into master

**Pull request merged**
